flexmeasures/ui/templates/crud/asset.html

```diff
@@ -7,38 +7,42 @@
 
 
 {% block divs %}
-<nav aria-label="breadcrumb">
-  <ol class="breadcrumb">
-    {% for breadcrumb in breadcrumb_info["ancestors"] %}
-      <li class="breadcrumb-item{% if loop.last %} dropdown active{% endif %}" {% if loop.last %}aria-current="page"{% endif %}>
-        {% if breadcrumb["url"] is not none and not loop.last %}
-          <a href="{{ breadcrumb['url'] }}">{{ breadcrumb['name'] }}</a>
-        {% else %}
-          <a class="dropdown-toggle" data-toggle="dropdown" href="#">{{ breadcrumb['name'] }}<span class="caret"></span></a>
-          <ul class="dropdown-menu">
-            {% for sibling in breadcrumb_info["siblings"] %}
-              <li><a href="{{ sibling['url'] }}">{{ sibling["name"] }}</a></li>
+
+    <nav class="mt-3 " aria-label="breadcrumb ">
+        <ol class="breadcrumb p-2">
+            {% for breadcrumb in breadcrumb_info["ancestors"] %}
+            <li class="breadcrumb-item{% if loop.last %} dropdown active{% endif %}" {% if loop.last %}aria-current="page"{% endif %}>
+                {% if breadcrumb["url"] is not none and not loop.last %}
+                <a  style="text-decoration:none; "href="{{ breadcrumb['url'] }}">{{ breadcrumb['name'] }}</a>
```

Comment:
Can we not add `text-decoration:none;` somehow to the styling of the `breadcrumb-item a` elements?

Also, your string should have a space afterwards (goes right into "href" here, I'm not sure how browsers are able to parse that)

Reply:
i have removed them, it was just for removing the underline in the links